### PR TITLE
Hyrax 880

### PR DESCRIPTION
**Pull request by kyang2014**

### CODE REVIEW

File: modules/hdf5_handler/HDF5RequestHandler.cc
Comment (jgallagher59701) [inline modules/hdf5_handler/HDF5RequestHandler.cc:626]: I think while this gets fixed (the order of these exceptions in the catch clauses) we should add `const` to the caught exception object declaration. So, "BESSyntaxError &e" becomes "const BESSyntaxUserError &e". The same change should be applied to the BESError &e declarations, too.

**Review by jgallagher59701 — CHANGES_REQUESTED** [review]
Add the 'const' qualifier to the exception declarations in the catch clauses and merge. Thanks.

File: modules/hdf5_handler/HDF5RequestHandler.cc
Comment (kyang2014) [reply modules/hdf5_handler/HDF5RequestHandler.cc]: Done.

**Comment by jgallagher59701** [comment]
Great. Once the tests pass - assuming they do - then merge. No need to ask for a second review.

Thanks.

**Pull request merged**
